src/WireMock.Net/Serialization/MappingConverter.cs

```diff
@@ -537,4 +555,85 @@ private static string FormatArray(JArray jArray, int ind)
 
         return $"new [] {{ {string.Join(", ", items)} }}";
     }
+
+    private static readonly HashSet<string> csharp_keywords = new HashSet<string>(new[]
```

Comment:
Can you move this to the top of the file?

(And maybe instead of using this list, you can maybe use:
https://github.com/StefH/ProxyInterfaceSourceGenerator/blob/main/src/ProxyInterfaceSourceGenerator/Extensions/SymbolExtensions.cs#L35
) ?

Reply:
Using `SyntaxFacts.GetKeywordKind` requires reference to `Microsoft.CodeAnalysis.CSharp`.
I can extract the whole logic responsible for generating anonymous object to a separate class, what fo you think?

Reply:
Yes, maybe a new internal static class, placed in `Util` folder.

Reply:
fixed

Reply:
Thanks